pwndbg/vmmap.py

```diff
@@ -266,7 +266,7 @@ def monitor_info_mem():
 
         pages.append(pwndbg.memory.Page(start, size, flags, 0, '<qemu>'))
 
-    return pages
+    return tuple(pages)
```

Comment:
This is to prevent `memoize`-related warnings that we are trying to hash mutable values.